modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -869,6 +875,10 @@ private CompletableFuture<Void> startPartitionAndStartClient(
                 }
 
                 if (((Loza) raftMgr).isStarted(raftNodeId)) {
+                    if (newAssignments != null && newAssignments.force()) {
```

Comment:
We have to perform "resetPastitions" on restart anyway, because groups can only start with persisted configuration, and this "reset" configuration will only be persisted after new leader is elected, which is not instantaneous.